### PR TITLE
[Docs] zh-cn document translate: IDE configuration

### PR DESCRIPTION
**Pull request by epiciskandar**
IDE configuration parts: Eclipse and VSCode

## Description

following after PR #14924

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* tracking issue: #15443

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: docs/zh-cn/other_eclipse.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# 在Eclipse中设置QMK开发环境
+*译注：由于译者没有Eclipse环境，部分菜单项可能与真实描述不同，请酌情甄别，并欢迎提cr进行修改完善*
```
Comment (peepeetee) [inline docs/zh-cn/other_eclipse.md:2]:
```suggestion
*译注：由于译者没有Eclipse环境，部分菜单项可能与真实描述不同，请酌情甄别，并欢迎提pr进行修改完善*
```

File: docs/zh-cn/other_eclipse.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+本文专注于阐述如何将Eclipse配置为AVR软件开发环境，并用于基于QMK代码的开发工作。
+
+注意，在本文编写时，仅在Ubuntu 16.04环境中进行过验证。
+
+# 需求
+## 构建环境
+在开始之前，你需要确保遵循了新手教程中的[新手指引](zh-cn/newbs_getting_started.md)一节。通常，此时你应该具备了[通过 `qmk complile` 命令](zh-cn/newbs_building_firmware.md#build-your-firmware)构建固件文件的能力。
+
+## Java
+Eclipse为Java程序，因此需要安装Java 8或更高版本才能允许。你可以选择JRE或JDK，后者在进行Java开发时需要用到。
```
Comment (peepeetee) [inline docs/zh-cn/other_eclipse.md:24]:
```suggestion
Eclipse为Java程序，因此需要安装Java 8或更高版本才能运行。你可以选择JRE或JDK，后者在进行Java开发时需要用到。
```

File: docs/zh-cn/other_eclipse.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+# 安装Eclipse及插件
+Eclipse有[多种可选安装方式](https://www.eclipse.org/downloads/eclipse-packages/)，取决于你的使用目标。目前没有完备的AVR开发栈安装包，所以我们需要从Eclipse CDT（C/C++ 开发工具环境）开始并安装对应的插件。
+
+## 下载安装Eclipse CDT
+如果系统中已安装了Eclipse CDT，可以跳过本步骤。同时，为了确保版本支持情况，我们推荐保持其更新至最新版。
+
+如果你已安装了Eclipse包，通常也可以[在上面再安装CDT插件](https://eclipse.org/cdt/downloads.php)。但是可能更好的方案是重新全新安装一下，以确保环境轻量，以及防止已安装的工具对后续的工程开发工作产生干扰。
+
+安装很简单：遵循[Eclipse安装5布走](https://eclipse.org/downloads/eclipse-packages/?show_instructions=TRUE)，并在第三步选择 **用于C/C++开发者的Eclipse IDE（Eclipse IDE for C/C++ Developers）**。
```
Comment (peepeetee) [inline docs/zh-cn/other_eclipse.md:34]:
```suggestion
安装很简单：遵循[Eclipse安装5步走](https://eclipse.org/downloads/eclipse-packages/?show_instructions=TRUE)，并在第三步选择 **用于C/C++开发者的Eclipse IDE（Eclipse IDE for C/C++ Developers）**。
```

**Review by peepeetee — CHANGES_REQUESTED** [review]

File: docs/zh-cn/other_eclipse.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
+# 在Eclipse中设置QMK开发环境
+*译注：由于译者没有Eclipse环境，部分菜单项可能与真实描述不同，请酌情甄别，并欢迎提pr进行修改完善*
```
Comment (peepeetee) [inline docs/zh-cn/other_eclipse.md:2]:
```suggestion

<!---
  original document: 0.15.12:docs/other_eclipse.md 
  git diff 0.15.12 HEAD -- docs/other_eclipse.md  | cat
-->

*译注：由于译者没有Eclipse环境，部分菜单项可能与真实描述不同，请酌情甄别，并欢迎提pr进行修改完善*
```

File: docs/zh-cn/other_vscode.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
+#  在Visual Studio Code中设置QMK开发环境
+
```
Comment (peepeetee) [inline docs/zh-cn/other_vscode.md:2]:
```suggestion

<!---
  original document: 0.15.12:docs/other_vscode.md 
  git diff 0.15.12 HEAD -- docs/other_vscode.md  | cat
-->

```

**Review by peepeetee — CHANGES_REQUESTED** [review]

File: docs/zh-cn/other_eclipse.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
+# 在Eclipse中设置QMK开发环境
+
+<!---
+  original document: 0.15.16:docs/other_eclipse.md 
+  git diff 0.15.16 HEAD -- docs/other_eclipse.md  | cat
+-->
+
+*译注：由于译者没有Eclipse环境，部分菜单项可能与真实描述不同，请酌情甄别，并欢迎提pr进行修改完善*
```
Comment (daskygit) [inline docs/zh-cn/other_eclipse.md:8]:
```suggestion

```
I wouldn't worry about including this line.

**Review by daskygit — CHANGES_REQUESTED** [review]

**Review by daskygit — APPROVED** [review]

**Review by peepeetee — APPROVED** [review]

**Comment by daskygit** [comment]
Thanks.

**Pull request merged**